iree/compiler/Dialect/VMLA/IR/VMLABase.td

```diff
@@ -160,7 +161,7 @@ class VMLA_ElementTypeOp<string mnemonic, list<OpTrait> traits = []> :
   }];
 }
 
-class VMLA_UnaryOp<string mnemonic, Attr typeAttr, list<OpTrait> traits = []> :
+class VMLA_UnaryOp<string mnemonic, Attr typeAttr, list<OpTrait> traits = []> : 
```

Comment:
Looks like you added trailing whitespace here. Please update these lines where you reverted your edits so they aren't unnecessarily modified.